src/test/java/com/ericsson/ei/frontend/WebControllerTest.java

```diff
@@ -51,7 +51,7 @@
     @Before
     public void beforeClass() throws EiBackendInstancesException {
         when(controllerUtils.getFrontEndServiceUrl()).thenReturn(FRONT_END_SERVICE_URL);
-        when(controllerUtils.getBackEndServiceUrl(any())).thenReturn(BACK_END_SERVICE_URL);
+        //when(controllerUtils.getBackEndServiceUrl(any())).thenReturn(BACK_END_SERVICE_URL);
```

Comment:
Why comment this out?

Reply:
Tested how it worked without it, and as it is not needed I should have removed it, but forgot. Will remove in next commit.

Reply:
Done.